test/base_middleware.test.ts

```diff
@@ -238,6 +238,67 @@ describe("BaseMiddleware", () => {
             done(err);
         });
     });
+
```

Comment:
I've just thought that the 'wrapping' of the service in the test above with `toDynamicValue` and `inRequestScope` should no longer be necessary. It was not documented in the README example.

I have amended the test to remove it locally and the test succeeds. Should I add that change to this PR?

Reply:
I've gone ahead and removed it now.